src/main/java/org/spongepowered/api/world/extent/BiomeArea.java

```diff
@@ -30,10 +30,31 @@
 
 /**
  * An area containing biomes.
+ *
+ * <p>Some methods accept a pair of two ints, representing the x and z location
+ * of a biome. Some other methods accept or return a {@link Vector2i}. The
+ * y position of this vector is equal to the z position of a biome in the
+ * world.</p>
  */
```

Comment:
I can certainly understand the reasoning behind this, although my concern would be that developers may become confused as to which y values represent z values and vice-versa.